mlflow/tracing/fluent.py

```diff
@@ -261,9 +263,9 @@ def get_current_active_span():
         The current active span if exists, otherwise None.
     """
     otel_span = trace_api.get_current_span()
-    if otel_span is None:
+    if otel_span is None or isinstance(otel_span, trace_api.NonRecordingSpan):
```

Comment:
note: OTel returns `NonRecordingSpan` if the tracer is not instantiated.

Reply:
Can we add this note into comment?